clarity_ext/data_cli.py

```diff
@@ -84,87 +86,29 @@ def get_stages(workflow_status, workflow_name, protocol_name, stage_name, use_ca
                 print("# ERROR workflow={}: {}".format(workflow.uri, e.message))
 
 
-@main.command("move-artifacts")
-@click.argument("artifact-name")
-@click.argument("unassign-stage-name")
+@main.command("move-artifacts-plan")
+@click.argument("artifact-ids", nargs=-1)
 @click.argument("assign-workflow-name")
 @click.argument("assign-stage-name")
-@click.option("--commit/--no-commit", default=False)
-def move_artifacts(artifact_name, unassign_stage_name, assign_workflow_name, assign_stage_name, commit):
-    """Moves all samples that are in a particular workflow from one workflow to another."""
-    # TODO: Currently removes it from all stages it's currently in and assigns it to only one
-    # TODO: This is a quick fix, so all the logic is currently in the CLI
-    from clarity_ext.service.routing_service import RerouteInfo, RoutingService
-
+def move_artifacts_plan(artifact_ids, assign_workflow_name, assign_stage_name):
+    """Creates a 'plan' for moving the artifact-id, reviewing all parameters etc.
+     This plan can be manually reviewed/edited and then executed by running move-artifacts-commit"""
```

Comment:
The move-artifacts main command use an option for commit, so the doc string and implementation is different

Reply:
Roger. I hope it's clearer now.